### PR TITLE
Tables Definition and GetList

### PR DESCRIPTION
**Pull request by ddcroit**

### CODE REVIEW

File: Backend/Platform/DataBase/AlpiUSDB/Functions/public_f_RefUNLOCO_GetList.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			ce."TableName" "CountryCodeExternalTableName",
+			sz."Code" "StandardZoneCode",
+			sz."TableName" "StandardZoneTableName",
+			dl."Code" "DaylightSavingZoneCode",
+			dl."TableName" "DaylightSavingZoneTableName"			
+		FROM
+			public."RefUNLOCO" r
+		LEFT JOIN
+			public."TimeZoneSet" tz
+				on r."TimeZoneSetId" = tz."PK"
```
Comment (DariaVasilenko) [inline Backend/Platform/DataBase/AlpiUSDB/Functions/public_f_RefUNLOCO_GetList.sql]: why left join? these columns are not nullable

**Comment by ddcroit** [comment]
Left Join removed.

**Comment by ddcroit** [comment]
@DariaVasilenko ; find a potential solution using arrays.

**Comment by ddcroit** [comment]
@DariaVasilenko Hello ! I have changed the param input per Maksim request.

**Pull request merged**